test/gtest/common/test_helpers.h

```diff
@@ -468,21 +468,36 @@ class message_stream {
 /**
  * Make uct_iov_t iov[iovcnt] array with pointer elements to original buffer
  */
-#define UCS_TEST_GET_BUFFER_IOV(_name_iov, _name_iovcnt, _buffer_ptr, _buffer_length, _memh, _iovcnt) \
+#define UCS_TEST_GET_BUFFER_IOV(_name_iov, _name_iovcnt, _buffer_ptr, _buffer_length, \
```

Comment:
This is macro because it creates `uct_iov_t local_iov[iovcnt]` on stack in particular function. Just to reduce the code lines in many places.